contribute/release-process.md

```diff
@@ -71,7 +71,7 @@ If you created a new branch, update the [CI - OWASP Dependency Check](https://gi
 
 Note that you should also stop the workflow for previous Pulsar versions that are EOL.
 
-Also, if you created a new branch, please update the "Supported Versions" table on the [version policy](version-policy.md) page. This table is for support timelines based on when minor releases take place.
+Also, if you created a new branch, please update the "Supported Versions" table on the [release policy](release-policy.md) page. This table is for support timelines based on when minor releases take place.
```

Comment:
Do we need to add instructions on how to update the "Supported Versions" table?

Reply:
It's now data-driven so the step updating `data/release-pulsar.js` should cover this requirement. I'll update the doc.

Reply:
Fixed by 88734f1d8af2b6a9d913452d996dc124a6a3de7e